Adl/Sign.hs

```diff
@@ -57,6 +57,18 @@ data Symbol
   | Rel Relation
     deriving (Eq, Ord, Show, Typeable, Data)
 
+data SymbolKind = ConK | RelK 
+  deriving (Eq, Ord, Show, Typeable, Data)
+
+symType :: Symbol -> SymbolKind
+symType (Con _) = ConK
+symType (Rel _) = RelK
```

Comment:
I think `symKind` is a better name for this function, because of the result type (and symbol types and kinds are distinguished e.g. in CASL).

Reply:
To do this, the function in Logic.Logic should be renamed to `sym_kind`, otherwise I get an error about ambiguities when defining `symKind Adl` in terms of `symKind`. Agreed?

Reply:
Actually it took less replacements to change `symType` with `sym_kind` for Adl, Maude and RelScheme.

Reply:
good